web/src/js/components/common.js

```diff
@@ -131,4 +131,26 @@ export var Splitter = React.createClass({
             </div>
         );
     }
-});
\ No newline at end of file
+});
+
+export class ToggleComponent extends React.Component{
```

Comment:
This component is great becase it does not have any state. We can just make it a pure component: https://facebook.github.io/react/docs/reusable-components.html#stateless-functions :smiley: